src/sql/workbench/parts/query/browser/queryActions.ts

```diff
@@ -605,11 +604,11 @@ export class ListDatabasesActionItem implements IActionViewItem {
 				actionLabel: nls.localize('listDatabases.toggleDatabaseNameDropdown', "Select Database Toggle Dropdown")
 			});
 			this._dropdown.onValueChange(s => this.databaseSelected(s));
-			this._toDispose.push(this._dropdown.onFocus(() => this.onDropdownFocus()));
+			this._register(this._dropdown.onFocus(() => this.onDropdownFocus()));
```

Comment:
This should probably be added to the registration as well

Reply:
What do you mean add to the registration?

Reply:
Guess this got put on the wrong line : 

```
this._dropdown.onValueChange(s => this.databaseSelected(s));
```

to

```
this._register(this._dropdown.onValueChange(s => this.databaseSelected(s)));
```

Reply:
O i see